ogr/services/gitlab/issue.py

```diff
@@ -58,13 +61,19 @@ def description(self) -> str:
         return self._raw_issue.description
 
     @property
+    @offline
     def author(self) -> str:
         return self._raw_issue.author["username"]
 
     @property
+    @offline
     def created(self) -> datetime.datetime:
         return self._raw_issue.created_at
```

Comment:
Are we sure it cannot touch the service? (Same for github.)

I have no problem with marking fewer properties offline just to be sure.

Reply:
I've marked those that definitely can't change, so in my opinion, if you create the Issue with `raw_issue` object, the `raw_issue` doesn't have to be updated nor queried to retrieve them...
However, it depends on the internal implementation of github/gitlab objects... I'll look into it when fixing the tests